src/rgw/rgw_sal_motr.cc

```diff
@@ -898,7 +901,9 @@ int MotrBucket::merge_and_store_attrs(const DoutPrefixProvider *dpp, Attrs& new_
   for (auto& it : new_attrs)
     attrs[it.first] = it.second;
 
-  return put_info(dpp, y, ceph::real_time());
+  // "merge_and_store_attrs" is meant to updating existing metadata, 
```

Comment:
why merge_and_store_attrs() needs to be mentioned here?

Reply:
Modifiedit to put_info